vendor/engines/split_accounts/app/services/split_accounts/reports/querier.rb

```diff
@@ -0,0 +1,29 @@
+module SplitAccounts
+
+  module Reports
+
+    class Querier < ::Reports::Querier
+
+      def perform
+        super.map do |order_detail|
+          simulate_order_details(order_detail)
+        end.flatten
+      end
+
+      def simulate_order_details(order_detail)
+        if order_detail.account.is_a?(SplitAccounts::SplitAccount)
```

Comment:
We have an `AccountExtension` module where we're adding `parent_split_accounts` and a couple other scopes/associations. We could add `splits` from there. But yeah, we definitely shouldn't put it directly in the Account class.